jest.config.js

```diff
@@ -178,10 +178,7 @@ module.exports = {
   // transform: undefined,
 
   // An array of regexp pattern strings that are matched against all source file paths, matched files will skip transformation
-  // transformIgnorePatterns: [
-  //   "/node_modules/"
-  // ],
-
+  transformIgnorePatterns: ['/node_modules/(?!(@sendbird)/)'],
```

Comment:
fixes jest test error

reference:
https://gist.github.com/bang9/cd03e74ec9b49687aec071b3dc7247c7

Reply:
Since this option operates based on the path, it's best to specify it explicitly when used in practice.

Reply:
/node_modules/@sendbird/
This should be it right?

Reply:
or is it something else? bc '/node_modules/(?!(@sendbird)/)' means everything under node_modules that is not @sendbird correct?

Reply:
<img width="858" alt="Screenshot 2024-02-29 at 11 22 06 AM" src="https://github.com/sendbird/sendbird-uikit-react/assets/16806397/e8c334b7-a5f2-417a-8155-58111b9cc3af">
Can you spot out which library should be transform ignored? based on the error log I thought `node_modules/@sendbird/react-uikit-message-template-view` is the one but that was not it.

Reply:
<img width="730" alt="Screenshot 2024-02-29 at 11 42 44 AM" src="https://github.com/sendbird/sendbird-uikit-react/assets/16806397/164e54bc-c284-42b8-a3e3-b793f776d0b6">
this solution suggested by ChatGPT did not work either

Reply:
```
  transformIgnorePatterns: [
    '/node_modules/(?!(?:@sendbird/(react-uikit-message-template-view|uikit-message-template))/)'
  ]
```
this solution worked. It excludes two packages from transformation:
react-uikit-message-template-view and uikit-message-template